contentcuration/contentcuration/static/js/edit_channel/publish/views/PublishWrapper.vue

```diff
@@ -2,25 +2,25 @@
 
   <VApp>
     <PublishModal />
-    <ProgressOverlay v-if="currentTask" />
+    <!-- <ProgressOverlay v-if="currentTask" /> -->
```

Comment:
Whenever commenting out code, I'd recommend putting a note about under which conditions it should be uncommented later. If there are no plans to reintroduce this code, then it's best to delete it, as leaving it in commented implies the idea is to undo the change later.

Reply:
Yep! I just left it out for now to quickly fix the linting errors and am planning on migrating the publish modal very soon (need to revisit the code to know how much I need to actually change)